README.md

```diff
@@ -190,22 +196,26 @@ There are two ways to specify the role: use the built-in role or create a custom
       - Microsoft.Compute/disks/beginGetAccess/action
       - Microsoft.Compute/disks/endGetAccess/action
    
-   Use the following commands to create a custom role which has the minimum required permissions:
+   Use the following commands to create a custom role which has the minimum required permissions (when subscriptionId and resourceGroup isn't set):
```

Comment:
Let's remove `(when subscriptionId and resourceGroup isn't set)` as we recommend users to use the new way and they may don't know what the `subscriptionId` and `resourceGroup`  are if they have no context about this change.